DataScience/dashboard_utils.py

```diff
@@ -98,11 +101,10 @@ def merge_and_unique_stats(stats_files, dashboard_file):
     print('Output dashboard data...')
     output_dashboard_data(d, dashboard_file)
 
-def create_stats(log_fp, d=None, predictions_files=None, is_summary=False, report_progress=True):
+def create_stats(log_fp, log_type='cb', d={}, predictions_files=None, is_summary=False, report_progress=True):
 
     t0 = time.time()
-    if d is None:
```

Comment:
this is a breaking change. Please revert since the behavior of python would be changed